src/test/kotlin/org/malachite/estella/interview/InterviewIntegration.kt

```diff
@@ -0,0 +1,318 @@
+import org.junit.jupiter.api.*
+import org.malachite.estella.BaseIntegration
+import org.malachite.estella.aplication.domain.ApplicationRepository
+import org.malachite.estella.commons.EStellaHeaders
+import org.malachite.estella.commons.models.interviews.Interview
+import org.malachite.estella.commons.models.offers.Application
+import org.malachite.estella.commons.models.offers.ApplicationStatus
+import org.malachite.estella.commons.models.offers.RecruitmentStage
+import org.malachite.estella.commons.models.offers.StageType
+import org.malachite.estella.commons.models.people.HrPartner
+import org.malachite.estella.commons.models.people.JobSeeker
+import org.malachite.estella.interview.api.MeetingDate
+import org.malachite.estella.interview.api.NotesFilePayload
+import org.malachite.estella.interview.domain.InterviewRepository
+import org.malachite.estella.people.domain.JobSeekerRepository
+import org.malachite.estella.people.seekers.DummyJobSeekerRepository
+import org.malachite.estella.process.domain.RecruitmentProcessRepository
+import org.malachite.estella.process.domain.RecruitmentStageRepository
+import org.malachite.estella.services.RecruitmentProcessService
+import org.malachite.estella.util.DatabaseReset
+import org.malachite.estella.util.jobSeekers
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.test.context.event.annotation.AfterTestExecution
+import org.springframework.test.context.event.annotation.BeforeTestMethod
+import strikt.api.expect
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+import strikt.assertions.isNull
+import java.sql.Date
+import java.sql.Timestamp
+import java.time.Instant
+import java.time.LocalDateTime
+import java.util.*
+import javax.sql.rowset.serial.SerialClob
+import kotlin.math.exp
+
+@DatabaseReset
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+class InterviewIntegration: BaseIntegration() {
+
+    @Autowired
+    private lateinit var interviewRepository: InterviewRepository
+    @Autowired
+    private lateinit var jobSeekerRepository: JobSeekerRepository
+    @Autowired
+    private lateinit var recruitmentStageRepository: RecruitmentStageRepository
+    @Autowired
+    private lateinit var applicationRepository: ApplicationRepository
+    @Autowired
+    private lateinit var recruitmentProcessRepository: RecruitmentProcessRepository
+
+    private lateinit var application: Application
+    private lateinit var jobseeker: JobSeeker
+    private lateinit var hrPartner: HrPartner
+
+    fun prepareApplication() {
+        val jobseeker = jobSeekerRepository.findAll().first()
+
+        val stage = recruitmentStageRepository.findAll().first()
+        hrPartner = recruitmentProcessRepository.findAll().first{ it.stages.contains(stage) }.offer.creator
+        val application = Application(null,
+                Date(Calendar.getInstance().time.time),
```

Comment:
Wszędzie korzystamy z timestampów, więc nie wiem, czy tu też legitne by to nie było, ale to trzeba sprawdzić, czy się teraz nie powaliłem.

Reply:
to nam chyba może zepsuć troche rzeczy ktore juz działają żeby to teraz zmieniac? XD i nie bardzo sie to ma do tego taska, proponuje wrzucic task na improvement